src/ExNavigationBar.js

```diff
@@ -240,11 +241,14 @@ export default class ExNavigationBar extends PureComponent {
       });
     });
 
+    let backgroundComponents = scenesProps.map(this._renderBackground, this);
+
     return (
       <View pointerEvents={this.props.visible ? 'auto' : 'none'} style={styles.wrapper}>
         {isTranslucent && <Components.BlurView style={[styles.translucentUnderlay, {height}]} />}
 
         <Animated.View style={containerStyle}>
+            {isTranslucent ? null : backgroundComponents}
```

Comment:
That's what we need to figure out? I think we may not because anyway we can do that as well !! Let me remove it.